bin/index.js

```diff
@@ -82,4 +88,4 @@ To run the system tests, go to either:
     HTTPS: https://localhost:${secureServerPort}/public/tester/system-tester.html
 
 Press ctrl+c at any time to exit debugging.
-`);
+`);
```

Comment:
So one of the linter rules is to always have an empty line at the end of your file (it's an old unix legacy problem where some tools expect it). If you're not seeing this as a lint error we should take a look at if your linter is working correctly.